.github/workflows/tag-boost-milestone.yml

```
@@ -0,0 +1,44 @@
+name: Tag Boost PRs with milestone
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  tag-pr:
+    runs-on: ubuntu-latest
+    if: |
+      github.event.pull_request.merged == true &&
+      contains(github.event.pull_request.labels.*.name, '[Plugin] Boost')
+    env:
+      MILESTONE_NAME: "boost/next"
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Create the milestone if it doesn't exist
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        REPO: ${{ github.repository }}
+      run: |
+        if ! gh api repos/${REPO}/milestones --jq '.[] | select(.title=="'"$MILESTONE_NAME"'")' | grep -q "$MILESTONE_NAME"; then
```

This could be done a bit more cleanly as something like
```suggestion
        if ! gh api "repos/${REPO}/milestones" | jq -e --arg NAME "$MILESTONE_NAME"  '.[] | select( .title == $NAME )' &>/dev/null; then
```
Using `jq`'s `-e` saves having to `grep`, and `--arg` saves having to worry about quotes in the variable when trying to embed it in the jq filter string.